internal/store/metadata.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2025 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	basemetrics "k8s.io/component-base/metrics"
+
+	metaapi "k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/kube-state-metrics/v2/pkg/metric"
+
+	generator "k8s.io/kube-state-metrics/v2/pkg/metric_generator"
+)
+
+var (
+	descAnnotationsHelp = "Kubernetes annotations converted to Prometheus labels."
+	descLabelsHelp      = "Kubernetes labels converted to Prometheus labels."
+	descCreationHelp    = "Unix creation timestamp"
```

Comment:
qq: why does it have 4 new metrics while the above only deletes 2 metrics (annotation and labels)?

Seems tests are passing, just want to see what happened.